runtime/onert/backend/gpu_cl/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+set(LIB_ONERT_BACKEND_GPU_CL onert_backend_gpu_cl)
+
+nnas_find_package(Opencl_Headers QUIET)
+if(NOT Opencl_Headers_FOUND)
+  return()
+endif(NOT Opencl_Headers_FOUND)
+
+nnas_find_package(Farmhash QUIET)
+if(NOT Farmhash_FOUND)
+  return()
+endif(NOT Farmhash_FOUND)
+
+nnas_find_package(Abseil QUIET)
+if(NOT Abseil_FOUND)
+  return()
+endif(NOT Abseil_FOUND)
+
+file(GLOB_RECURSE SOURCES "*.cc")
+
+
+add_library(${LIB_ONERT_BACKEND_GPU_CL} SHARED ${SOURCES})
+
+target_include_directories(${LIB_ONERT_BACKEND_GPU_CL} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+target_include_directories(${LIB_ONERT_BACKEND_GPU_CL} PRIVATE ${NNAS_EXTERNALS_DIR}/OPENCL_HEADERS/)
```

Comment:
Instread of using explicit path like `${NNAS_EXTERNALS_DIR}/OPENCL_HEADERS`, we can use more general path like `${Opencl_Headers_DIR}/OPENCL_HEADERS`